docker/opensearch/opensearch-dashboards/Dockerfile.j2

```diff
@@ -10,7 +10,20 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build
 {{ macros.configure_user(name='opensearch-dashboards', shell='/bin/bash', homedir='/usr/share/opensearch-dashboards') }}
 {{ macros.enable_extra_repos(['opensearch-dashboards']) }}
 
-{% set opensearch_dashboards_packages = ['opensearch-dashboards'] %}
+# NOTE(mgoddard): Pinning to 2.15 due to https://bugs.launchpad.net/kolla/+bug/2077319
```

Comment:
surely it will downgrade current opensearch in our caracal images to v2.16.0 
<img width="121" alt="image" src="https://github.com/user-attachments/assets/2ff08992-d1d1-4c32-99a8-b216848f4d39">

Reply:
Looks like a fix has merged upstream, not sure if it has released yet: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8082

Reply:
Is this SMS Lab? Are we experiencing the discover page bug?

Reply:
2.17.0 was released 3 days ago, I don't like x.0 releases but we could bump to that

Reply:
> Is this SMS Lab? Are we experiencing the discover page bug?

we already had a index pattern (flog-*), so we didn't hit that

> 2.17.0 was released 3 days ago, I don't like x.0 releases but we could bump to that

So we should remove pin upstream and test if this is fixed.

Reply:
I'll take w look into that

Reply:
it is in 2.17 https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8097

Reply:
https://review.opendev.org/c/openstack/kolla/+/930685